pageStyles/testpage.module.scss

```diff
@@ -0,0 +1,66 @@
+﻿@import "constants";
```

Comment:
Could these styles be moved to be near the components rather than in the general location?

(ie these look like styling for the component rather than for the whole page)

Reply:
done

Reply:
Doesn't seem to have moved yet.

With modules, it should be the case that you've got a file system like

```txt
components
    - ComponentA
        - ComponentA.tsx
        - ComponentA.module.scss
    - ComponentB
        - ComponentB.tsx
        - ComponentB.module.scss
```

at the moment, we've got 2 problems
1) this file is in `pageStyles`,rather than near the thing it is styling.
2) this includes styles for lots of components - each component should have its own css module, containing just the stuff it needs.

Reply:
doesn't appear to have been resolved just yet?